src/sentry/incidents/endpoints/organization_alert_rule_activity_details.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework.response import Response
+
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.alert_rule import DetailedAlertRuleActivitySerializer
+from sentry.incidents.endpoints.bases import OrganizationAlertRuleEndpoint
+
+
+class OrganizationAlertRuleActivityDetailsEndpoint(OrganizationAlertRuleEndpoint):
```

Comment:
My thought here was that we'd just have an endpoint that returns all the activities for an alert, rather than one that returns everything. I'm not totally sure on your data requirements here, how much detail do you need about each incident?

Reply:
Need some of the incident data to show in the timeline (`dateStarted`, `dateClosed`, `statusMethod`, `seenBy` list) as well as the activities grouped by incident